components/elm/src/biogeochem/AllocationMod.F90

```diff
@@ -1538,7 +1523,7 @@ subroutine Allocation2_ResolveNPLimit (bounds, num_soilc, filter_soilc  , &
                         endif
 
                         plant_pdemand_vr_patch(p,j) = vmax_plant_p(ivt(p)) * frootc(p) * froot_prof(p,j) * &
-                             cp_scalar(p) * t_scalar(c,j) * compet_plant_p(p)
+                             cp_scalar_runmean(p) * t_scalar(c,j) * compet_plant_p(p)
```

Comment:
@qzhu-lbl @rgknox This fix is needed in master

Reply:
yes